.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: c
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+script: bash -ex .travis-docker.sh
+services:
+- docker
+env:
+  global:
+  - PINS="cbor:."
```

Comment:
Why is this PINS needed? From README-travis I would think it is to bring in dependencies?